bigtable-hbase-parent/bigtable-hbase-mapreduce/src/main/java/com/google/cloud/bigtable/mapreduce/ShuffledSequenceFileInputFormat.java

```diff
@@ -0,0 +1,23 @@
+package com.google.cloud.bigtable.mapreduce;
```

Comment:
Please add the standard copyright header we have in most of our classes.  It should be set to 2016 for new files.